lms/djangoapps/courseware/grades.py

```diff
@@ -255,11 +350,14 @@ def create_module(descriptor):
 
     letter_grade = grade_for_percentage(course.grade_cutoffs, grade_summary['percent'])
     grade_summary['grade'] = letter_grade
-    grade_summary['totaled_scores'] = totaled_scores  	# make this available, eg for instructor download & debugging
+    grade_summary['totaled_scores'] = totaled_scores    # make this available, eg for instructor download & debugging
     if keep_raw_scores:
         # way to get all RAW scores out to instructor
         # so grader can be double-checked
         grade_summary['raw_scores'] = raw_scores
+
+    max_scores_cache.push_to_remote()
```

Comment:
Yeah, that occurred to me, but the indentation on the grading code is already 5-6 levels deep, and I didn't want to add a level.